<?xml version="1.0"?>
<launch>
    
    <node name="model_test_node" pkg="box_detector" type="test_node" output="screen">
        <param name="distance_max" type="double" value = "0.96500075" />
        <param name="distance_min" type="double" value = "0" />
    </node>
    
    <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 0 0 1 map model 100" />

</launch>